python/tvm/contrib/hexagon/session.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         remote_kw: dict,
         session_name: str = "hexagon-rpc",
         remote_stack_size_bytes: int = 256 * 1024,  # Min size for main thread in QuRT/sim
-        rpc_receive_buffer_size_bytes: int = 5 * 1024 * 1024,  # Size for passing hexagon tests
+        rpc_receive_buffer_size_bytes: int = 1024 * 1024 * 1024,  # Size for passing hexagon tests
```

Comment:
Left over from testing? A gigabyte for the rpc buffer can impact available memory for model execution